.github/actions/install-dependencies/action.yml

```diff
@@ -1,19 +1,33 @@
 name: "Install Dependencies"
 description: "Install dependencies, fetching from cache when possible"
+inputs:
+  node-version:
+    description: the version of Node.js to install
+    default: 18.17.1
 runs:
   using: "composite"
   steps:
     - name: Install pnpm
       uses: pnpm/action-setup@v2
       with:
-        version: 8.8.0
+        version: 9.1.1
 
-    - name: Install Node.js ${{ env.node-version }}
+    - name: Install Node.js ${{ inputs.node-version }}
       uses: actions/setup-node@v3
       with:
-        node-version: ${{ env.node-version }}
+        node-version: ${{ inputs.node-version }}
         cache: "pnpm"
 
+    - name: "Create .env file"
```

Comment:
Could we rename this as we have in other actions (I think it was to something like "Configure turbo repo cache environment variables")

Reply:
Absolutely!!